css/BadgeOverlaySupport.less

```diff
@@ -0,0 +1,8 @@
+.moon-badge-overlay-support-scrim {
+	display: block;
+	z-index: 1000;
+	width: 100%;
+	height: 55px;
+	color: white;
+	background-color: rgba(0, 0, 0, 0.5);
+}
```

Comment:
Please extract each value and add those values to ```moonstone-variables.less``` (or ```moonstone-variables-dark.less``` and ```moonstone-variables-light.less```)
You can refer to SelectionOverlaySupport.less file